src/backend/expungeservice/models/disposition.py

```diff
@@ -1,9 +1,14 @@
-from datetime import datetime
+from dataclasses import dataclass
+from datetime import date, datetime
+from typing import Optional
 
 
+@dataclass(eq=False)
```

Comment:
Same question here as the other PR. Why label this as a data class, but treat it as a regular class?

Reply:
Hmm maybe we have a conceptual difference of what a data class is. A data class is still just a regular Python class. From a functional perspective, you can view the dataclass class wrapper as a pattern that adds `__repr__`, `__init__` (including removing boilerplate), and `__eq__` to the class. Optionally it can include `__hash__` and ordering methods. I'll also say that tells the code reader that this class is primarily about holding data.

Reply:
Yes, we have a different conceptual difference. I re-read PEP 557, and I now understand why they chose to do it this way rather than the way that I would expect. For me a dataclass would be or less like a tuple. 

> Such a class is called a Data Class, but there's really nothing special about the class: the decorator adds generated methods to the class and returns the same class it was given.

Reply:
Hmm, so to be clear, are you ok with this `@dataclass(eq=False)` change now or not?

Reply:
Yes

Reply:
I still don't like it, but I'm OK with it since it seems to be the python way.

Reply:
Thanks for being patient and helping me learn!

Reply:
I've learned a bunch myself and I'm sure these conversations will be useful to anyone working with this repo in the future.